daemons/gptp/common/ptp_message.cpp

```diff
@@ -136,30 +136,29 @@ PTPMessageCommon *buildPTPMessage
 		if (ts_good != GPTP_EC_SUCCESS) {
 			char msg[HWTIMESTAMPER_EXTENDED_MESSAGE_SIZE];
 			port->getExtendedError(msg);
-			XPTPD_ERROR
+			GPTP_LOG_ERROR
 			    ("*** Received an event packet but cannot retrieve timestamp, discarding. messageType=%u,error=%d\n%s",
 			     messageType, ts_good, msg);
 			//_exit(-1);
```

Comment:
I dont this code was already here, but it looks like dead code to me. Should we remove this line?